app/assets/v2/js/pages/increase_bounty.js

```diff
@@ -0,0 +1,160 @@
+load_tokens();
+var setUsdAmount= function (event) {
```

Comment:
itd be great to DRY this (maybe move it into shared.js)

Reply:
On it.